include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -166,7 +166,7 @@ class argument_parser
      * \param[in] argc     The number of command line arguments.
      * \param[in] argv     The command line arguments to parse.
      */
-    argument_parser(std::string const app_name, int const argc, const char ** argv)
+    argument_parser(std::string const app_name, int const argc, char const * const * argv)
```

Comment:
`char const * const * const argv`
This would also cover const arrays of const arrays of const char, I think.